noaa/operations/default.json

```diff
@@ -181,5 +181,537 @@
           }
         }
       }
+    },
+    {
+      "name": "max_temp",
+      "operation-type": "search",
+      "body": {
+        "size": 0,
+        "query": {
+          "range": {
+            "station.elevation": {
+              "gt": 1000
+            }
+          }
+        },
+        "aggs": {
+          "max_temp": {
+            "max": {
+              "field": "TMAX"
+            }
+          }
+        }
+      }
+    },
+    {
+      "name": "last_max_temp_top_hits",
+      "operation-type": "search",
+      "body": {
+        "size": 0,
+        "query": {
+          "range": {
+            "station.elevation": {
+              "gt": 1000
+            }
+          }
+        },
+        "aggs": {
+          "last_temp": {
+            "top_hits": {
+              "size": 1,
+              "sort": {"date": "desc"},
+              "stored_fields": "_none_",
+              "docvalue_fields": ["TMAX"]
+            }
+          }
+        }
+      }
+    },
+    {
+      "name": "last_max_temp_top_metrics",
+      "operation-type": "search",
```

Comment:
Question:

I ran this manually using:

```
./rally --track-path=/Users/dl/source/elastic/rally-tracks/noaa --challenge="top_metrics" --car="4gheap,basic-license" --on-error=abort --client-options="timeout:240,request_timeout:240" --distribution-version=7.6.0 --runtime-jdk=bundled
```

In this task I received a parser not found error:

```
Running delete-index                                                           [100% done]
Running create-index                                                           [100% done]
Running check-cluster-health                                                   [100% done]
Running index                                                                  [100% done]
Running refresh-after-index                                                    [100% done]
Running force-merge                                                            [100% done]
Running refresh-after-force-merge                                              [100% done]
Running max_temp                                                               [100% done]
Running last_max_temp_top_hits                                                 [100% done]
Running last_max_temp_top_metrics                                              [  0% done]
[ERROR] Cannot race. Error in load generator [0]
	("Request returned an error. Error type: transport, Description: named_object_not_found_exception ({'error': {'root_cause': [{'type': 'named_object_not_found_exception', 'reason': '[1:98] unable to parse BaseAggregationBuilder with name [top_metrics]: parser not found'}], 'type': 'named_object_not_found_exception', 'reason': '[1:98] unable to parse BaseAggregationBuilder with name [top_metrics]: parser not found'}, 'status': 400})", None)
```

Is also tried with Elasticsearch master and got the same error.

Reply:
`top_metrics` landed in 7.7.0. When you tried master did you run with the basic license? If so *that* is strange. I'll check it on my side.

Reply:
Yes I tried with `basic` license and `master.

Reply:
I just tried in 7.7 and didn't get the error. Are you super sure you ran with the basic license.

Reply:
Checking again

Reply:
Oh ho!
```
[WARNING] No throughput metrics available for [last_max_temp_top_metrics]. Likely cause: Error rate is 100.0%. Please check the logs.
```

Maybe I've confused myself.... Sorry!

Reply:
The error is different actually:
```
{"error":{"root_cause":[{"type":"x_content_parse_exception","reason":"[20:13] [top_metrics] unknown field [metrics] did you mean [metric]?"}],"type":"x_content_parse_exception","reason":"[20:13] [top_metrics] unknown field [metrics] did you mean [metric]?"},"status":400}
```

Reply:
That means it doesn't have https://github.com/elastic/elasticsearch/pull/53163

Reply:
And *that* goes away when I pull in the latest 7.x. Which makes sense. ok. That was just a problem on my end.

Reply:
I think I found the culprit for the missing `top_metrics` parser. The git history state prevented an update to the latest ES master, so was testing against the master from some weeks ago.

My new test run has now passed `last_max_temp_top_metrics`, so I think we are good, sorry for the false alarm.